src/rgw/rgw_op.cc

```diff
@@ -7094,7 +7094,7 @@ void RGWDeleteMultiObj::execute(optional_yield y)
         aio_count--;
       }); 
     } else {
-      handle_individual_object(obj_key, y, &*formatter_flush_cond);
+      handle_individual_object(obj_key, y, nullptr);
```

Comment:
what about the case that we got here because `max_aio = 1`?
the `formatter_flush_cond` would not be null

Reply:
thanks, i removed the `max_aio > 1` condition above